aries-site/src/tests/navigation/deeplinks.js

```diff
@@ -1,8 +1,8 @@
-
+/* eslint-disable max-len */
```

Comment:
I don't think we should disable it, just wrap it on the next line. easy fix.

Reply:
I can, thats not a problem but if thats the case then we should change that in all the testing files because alot of our descriptions were past 80characters so pretty much every file has to  
`/* eslint-disable max-len */`
to stay consistent I would change that in all of them

Reply:
I understand. we should make sure there are no new `eslint-disable max-len`  on top of the files and change the other files gradually, in the meanwhile we should focus on fixing this file properly.